packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts

```diff
@@ -32,197 +40,151 @@ Chains not supported yet due fireblocks or zerodev doesn't supported
+        ["ethereum-sepolia", "https://ethereum-sepolia.publicnode.com"],
+        ["mumbai", "https://rpc-mumbai.maticvigil.com"],
+        ["zkatana", "https://rpc.startale.com/zkatana"],
     ]).get(chain)!;
 }
 
-export function getBlockExplorerByBlockchain(chain: Blockchain) {
+export function getBlockExplorerByBlockchain(chain: BlockchainIncludingTestnet) {
     return new Map([
-        [Blockchain.ETHEREUM, "https://etherscan.io"],
-        [Blockchain.POLYGON, "https://polygonscan.com"],
-        [Blockchain.BSC, "BNB_BSC"],
-        [Blockchain.OPTIMISM, "https://optimistic.etherscan.io"],
-        [Blockchain.ARBITRUM, "https://arbiscan.io"],
-        [Blockchain.GOERLI, "https://goerli.etherscan.io"],
-        [Blockchain.SEPOLIA, "https://sepolia.etherscan.io"],
-        [Blockchain.MUMBAI, "https://mumbai.polygonscan.com"],
-        [Blockchain.ZKATANA, "https://zkatana.explorer.startale.com"],
+        ["ethereum", "https://etherscan.io"],
+        ["polygon", "https://polygonscan.com"],
+        ["bsc", "BNB_BSC"],
+        ["optimism", "https://optimistic.etherscan.io"],
+        ["arbitrum", "https://arbiscan.io"],
+        ["goerli", "https://goerli.etherscan.io"],
+        ["ethereum-sepolia", "https://sepolia.etherscan.io"],
+        ["mumbai", "https://mumbai.polygonscan.com"],
+        ["zkatana", "https://zkatana.explorer.startale.com"],
     ]).get(chain)!;
 }
 
-export function getDisplayNameByBlockchain(chain: Blockchain) {
+export function getDisplayNameByBlockchain(chain: BlockchainIncludingTestnet) {
     return new Map([
-        [Blockchain.ETHEREUM, "Ethereum Mainnet"],
-        [Blockchain.POLYGON, "Polygon Mainnet"],
-        [Blockchain.BSC, "BNB_BSC"],
-        [Blockchain.OPTIMISM, "Optimism"],
-        [Blockchain.ARBITRUM, "Arbitrum"],
-        [Blockchain.GOERLI, "Goerli Tesnet"],
-        [Blockchain.SEPOLIA, "Sepolia Tesnet"],
-        [Blockchain.MUMBAI, "Mumbai Tesnet"],
-        [Blockchain.ZKATANA, "zKatana Tesnet"],
+        ["ethereum", "Ethereum Mainnet"],
+        ["polygon", "Polygon Mainnet"],
+        ["bsc", "BNB_BSC"],
+        ["optimism", "Optimism"],
+        ["arbitrum", "Arbitrum"],
+        ["goerli", "Goerli Tesnet"],
+        ["ethereum-sepolia", "Sepolia Tesnet"],
+        ["mumbai", "Mumbai Tesnet"],
+        ["zkatana", "zKatana Tesnet"],
     ]).get(chain)!;
 }
 
-export function getTickerByBlockchain(chain: Blockchain) {
+export function getTickerByBlockchain(chain: BlockchainIncludingTestnet) {
     return new Map([
-        [Blockchain.ETHEREUM, "ETH"],
-        [Blockchain.POLYGON, "MATIC"],
-        [Blockchain.BSC, "BNB_BSC"],
-        [Blockchain.OPTIMISM, "OP"],
-        [Blockchain.ARBITRUM, "ARB"],
-        [Blockchain.GOERLI, "ETH"],
-        [Blockchain.SEPOLIA, "ETH"],
-        [Blockchain.MUMBAI, "MATIC"],
-        [Blockchain.ZKATANA, "ETH"],
+        ["ethereum", "ETH"],
+        ["polygon", "MATIC"],
+        ["bsc", "BNB_BSC"],
```

Comment:
I think this should be just `"BNB"`
```suggestion
        ["bsc", "BNB"],
```